scipy/spatial/tests/test_distance.py

```diff
@@ -1043,6 +1079,39 @@ def test_pdist_dtype_equivalence(self):
                         y2 = pdist(new_type(X1), metric=metric)
                         _assert_within_tol(y1, y2, eps, verbose > 2)
 
+    def test_pdist_out(self):
+        # Test that out parameter works properly
+        eps = 1e-07
+        for metric in _METRICS_NAMES:
+            for rnd_eo_name in self.rnd_eo_names:
```

Comment:
Just tried, and unfortunately, `eo['cdist-X1']` has too few observations for mahalanobis distance. I might have to make a new data set, 10 x 9 or something like that.